modules/nanopb/Kconfig

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2021 Basalte bv
+# SPDX-License-Identifier: Apache-2.0
+
+config ZEPHYR_NANOPB_MODULE
+	bool
+
+menuconfig NANOPB
+	bool "Nanopb Support"
+	help
+	  This option enables the Nanopb library and generator.
+
+if NANOPB
+
+config NANOPB_ENABLE_MALLOC
+	bool "Enable malloc usage"
+	help
+	  This option enables dynamic allocation support in the decoder.
+
+config NANOPB_MAX_REQUIRED_FIELDS
+	int "Max number of required fields"
+	default 64
+	help
+	  Maximum number of proto2 required fields to check for presence.
+	  Default value is 64. Compiler warning will tell if you need this.
```

Comment:
... if you need **to change** this. ?
or
... if you need **to increase** this. ?

Reply:
I used the same documentation as the original repo, but it is actually just the minimum value, and the compiler will complain if it is set lower than 64.

I will update the description.